salt/modules/htpasswd.py

```diff
@@ -139,3 +139,46 @@ def userdel(pwfile, user, runas=None):
     out = __salt__['cmd.run'](cmd, runas=runas,
                               python_shell=False).splitlines()
     return out
+
+
+def verify(pwfile, user, password, opts='', runas=None):
+    '''
+    Return True if the htpasswd file exists, the user has an entry, and their
+    password matches.
+
+    pwfile
+        Path to htpasswd file
+
+    user
+        User name
+
+    password
+        User password
+
+    opts
+        Valid options that can be passed are:
```

Comment:
Instead of doing this, I think it might be better to split these out into individual keyword arguments, such as `md5=True`, etc, etc.

There are two reasons for this. First, I think it might be easier for somebody just looking at the function arguments to understand, and secondly because Salt reserves the keyword `opts` in many places to refer to the Salt options dictionary. It might be too easy to confuse the two if it's used here.